src/main/java/org/opensearch/security/configuration/ConfigurationRepository.java

```diff
@@ -299,7 +312,7 @@ private void waitForSecurityIndexToBeAtLeastYellow() {
                 response == null ? "no response" : (response.isTimedOut() ? "timeout" : "other, maybe red cluster")
             );
             try {
-                Thread.sleep(500);
+                TimeUnit.MICROSECONDS.sleep(500);
```

Comment:
Previously this was in milliseconds - this will increase the load on the cluster manager when its trying to get into a healthy state, why the change?

Reply:
good catch. my bad did not notice